src/batlabmainwindow.cpp

```diff
@@ -477,8 +492,69 @@ void BatlabMainWindow::showNewCellPlaylistWizard() {
     wizard->show();
 }
 
+cellResultsDisplayInfo createInitializedcellResultsDisplayInfo() {

```

Comment:
Thoughts on moving this into Batlablib as it is the sort of thing we could re-use in a command line or other application?

Reply:
Good call! I moved it.